.github/workflows/continuous-integration.yml

```diff
@@ -10,63 +10,69 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
```

Comment:
Why using the matrix strategy here and elsewhere besides the unit tests? If the purpose is easy parameterization then I think setting a global environment variable (https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#env) would be better suited since running these steps on multiple versions is not necessary. This solution also has the advantage of being global so there is no need to define it multiple times.

Reply:
Using a matrix does not bury `7.4` into steps.
It is like a constant in a PHP class.

Reply:
Yes, I understand, although I don't really mind it since I usually use search and replace. What is the problem with the  environment variable based approach?

Reply:
environment variable are general solutions for *everything* - similar to global variables in PHP

the matrix is specifically for e.g. PHP version